app/controllers/works_controller.rb

```diff
@@ -138,7 +138,7 @@ def collected
     options[:page] = params[:page] || 1
     options[:show_restricted] = current_user.present? || logged_in_as_admin?
 
-    @user = User.find_by(login: params[:user_id])
+    @user = User.find_by!(login: params[:user_id])
 
     return unless @user.present?
```

Comment:
I believe the `find_by!` in line 141 means that `@user.present?` is guaranteed to be true at this point. So you should be able to delete this line now.